docs/core/deploying/trimming/incompatibilities.md

```diff
@@ -29,6 +35,10 @@ Trimming and dynamic assembly loading is a common problem for systems that suppo
 
 The following sections list known incompatibilities with trimming on Windows.
 
+### NET programming with C++/CLI
+
+[NET programming with C++/CLI](/cpp/dotnet/dotnet-programming-with-cpp-cli-visual-cpp) is not supported with trimming.
```

Comment:
@agocke @sbomer @vitek-karas - I thought I remember seeing something about supporting C++/CLI , but I can't remember. Basically I thought the trimmer would scan the CLI image, but not trim anything inside it.

Reply:
It depends how the C++/CLI is used. If you call the assembly from managed, it might work (we copy it untouched, and scan it, as you correctly noted). But if you first call it from native, I doubt that will work - we would need the native hosting side of things to kick in, which is not trim compatible (at least not by default).
So I think it's better to simply state that it's not supported... if nothing else, we also don't have any tests for this currently. If we think we should support at least that one direction, then we should add tests for it somewhere. (We do test the trimming part, but I'm pretty sure we don't test it actually working after trimming).

Reply:
I could write.

is not ***CURRENTLY*** supported. It's possible to call an assembly from managed, but that approach is not supported. Calling C++/CLI from native is incompatible with trimming. 

(Emphasis is used to see change, won't be used in the doc).

Or keep as is, the conservative approach. 

I just don't want to stall folks who'd like to eventually migrate to .NET code from not getting started sooner.

Reply:
Personally I would prefer we say we don't support it (currently). I'm not confident enough to say that we did the work to support it.